frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java

```diff
@@ -161,12 +160,13 @@ public void initChannel(Channel ch) {
     }
 
     public boolean sendLoadMessage(CountDownLatch latch) {
+        int gpu = this.gpuId;
         try {
             latch.await();
             Job job = new Job(null, "load", new Payload(null, ""));
             model.addFirst(job);
         } catch (InterruptedException e) {
-            logger.warn("Backend worker thread interrupted.", e);
+            logger.warn("Backend worker thread interrupted to start in gpu {}.", gpu, e);
```

Comment:
this will not print out stacktrace. change to: logger.warn("Backend worker thread interrupted to start in gpu: " + gpu, e);